server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -259,34 +269,103 @@ private TrTag renderTIRow(AccountModel ti) {
-            "border-b",
-            "border-gray-300",
-            StyleUtils.even("bg-gray-100"))
-        .with(renderInfoCell(applicant))
-        .with(renderApplicantInfoCell(applicant))
-        .with(renderActionsCell(applicant))
-        .with(renderDateOfBirthCell(applicant))
-        .with(renderUpdateClientInfoCell(applicant.id));
+  private LiTag renderClientCard(AccountModel account) {
+    return li().withClass("usa-card tablet-lg:grid-col-6 widescreen:grid-col-4")
+        .with(
+            div()
+                .withClass("usa-card__container")
+                .with(
+                    div()
+                        .withClasses("usa-card__header", "flex justify-between")
+                        .with(
+                            div(
+                                    div(
+                                            renderSubHeader(account.getApplicantName())
+                                                .withClass("usa-card__heading"),
+                                            u(renderUpdateClientLink(account.id)).withClass("ml-2"))
+                                        .withClass("flex"),
+                                    renderCardDateOfBirth(account))
+                                .withClass("flex-col"),
+                            renderIndexPageLink(account)),
+                    div()
+                        .withClasses("usa-card__body", "flex")
+                        .with(
+                            renderCardContactInfo(account).withClasses("w-2/5"),
+                            renderCardApplications(account).withClasses("ml-10 w-2/5"),
+                            renderCardNotes(account.getTiNote()).withClasses("ml-10 w-3/5"))));
+  }
+
+  private DivTag renderCardContactInfo(AccountModel account) {
+    Optional<ApplicantModel> newestApplicant = account.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return div();
+    }
+    ApplicantData applicantData = newestApplicant.get().getApplicantData();
+    Optional<String> maybePhoneNumber = applicantData.getPhoneNumber();
+    String email = account.getEmailAddress();
+
+    return div(
+        label("Contact information").withFor("card_contact_info").withClass("whitespace-nowrap"),
+        div()
+            .condWith(
+                maybePhoneNumber.isPresent(),
+                div(
+                        Icons.svg(Icons.PHONE).withClasses("h-3", "w-3", "mr-1"),
+                        p(formatPhone(maybePhoneNumber.orElse(""))))
+                    .withClass("flex items-center"))
+            .condWith(
+                email != null && !email.isEmpty(),
+                div(Icons.svg(Icons.EMAIL).withClasses("h-3", "w-3", "mr-1"), p(email))
+                    .withClass("flex items-center"))
+            .withClass("text-xs")
+            .withId("card_contact_info"));
+  }
+
+  private String formatPhone(String phone) {
+    return phone.replaceFirst("(\\d{3})(\\d{3})(\\d+)", "($1) $2-$3");
```

Comment:
I think we can reuse the library for uniform phone number formatting. Please check- https://github.com/civiform/civiform/blob/6cb63fd6bbbfb1308b63fe3b20f331ba610ed0ad/server/app/services/applicant/question/PhoneQuestion.java#L98

Reply:
Yes, that will work.  I'm able to use the Google `PhoneNumberUtil` if I use the National format.  Making that change now.